### DISCUSSION TITLE
Add support for new ARRI ALF-4 look file format

### DISCUSSION THREAD

**Discussion by geoffsmithBK** [videovillage/community#2] [Ideas]
Would be great to be able to convert an existing 3D LUT into an ALF-4/.alf4 look file for quick/direct import into Alexa 35 as with the existing support for ALF-2/.aml. ARRI's white paper on ALF-4 is here:

https://www.arri.com/resource/blob/283114/259480ccaaeccc21483ba1180f72c802/2022-07-arri-alexa-35-arri-look-file-alf4-data.pdf

(however it's a binary format and there's no spec. included in the document)

Cheers,
--Geoff

**Comment by r2tincan** [comment]
Just chiming in to +1 this, came here to post. A LogC 3 -> LogC 4 LUT approximation for both .cube with the rec709 and LogC3 .cube - > ALF4 would be really awesome too. Help us with LogC4 lol

**Comment by gregcotten** [comment]
ARRI's wheel reinvention has caused more trouble than it's worth! We'll add rudimentary support (just supporting the normal LogC4 to LogC4 ALF4). I'll look into "converting" looks into the LogC4 container, but without the math for ARRI's V4 creative tone mapping + colorspace conversions I'm hesitant to release anything with that kind of support.

**Reply by VicoRender** [reply]
+1 for all of this. But please, also consider a .alf4 import.
Thanks!!!

**Reply by addit248** [reply]
+1 again, missing this feature in Lattice. Color graders aren't updating their workflow yet.
